storage/innobase/btr/btr0sea.cc

```diff
@@ -101,6 +108,32 @@ btr_search_get_n_fields(
 	return(n_fields + (n_bytes > 0 ? 1 : 0));
 }
 
```

Comment:
May be better to use a macro turning into nothing in release mode
something like
```c++
#ifdef UNIV_DEBUG
  #define btr_search_mem_accounting_validate() btr_search_mem_accounting_validate_impl()
#else
  #define btr_search_mem_accounting_validate()
#endif
static void btr_search_mem_accounting_validate_impl(void) { ... }
```